src/kibana/plugins/vis_types/vislib/editors/tile_map.html

```diff
@@ -10,6 +10,13 @@
   </select>
 </div>
 
+<div class="form-group">
+  <label>
+    <input type="checkbox" ng-model="vis.params.addLeafletPopup">
```

Comment:
Use the standard `vis.params.addTooltip` instead of `vis.params.addLeafletPopup`